support/safe_intrinsics/src/lib.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Provides a safe wrapper around some CPU instructions.
+//!
+//! This is needed because Rust's intrinsics are marked unsafe (despite
+//! these few being completely safe to invoke).
+
+#![no_std]
+// UNSAFETY: Calling a cpu intrinsic.
+#![allow(unsafe_code)]
+
+/// Invokes the cpuid instruction with input values `eax` and `ecx`.
+#[cfg(target_arch = "x86_64")] // xtask-fmt cpu-intrinsic
```

Comment:
you're almost there - now that you've added an exception to this crate, you can remove all of these // xtask-fmt annotations

Reply:
Don't we still need them for intrinsics?